osf/models/metaschema.py

```diff
@@ -39,7 +39,7 @@ def allow_egap_admins(queryset, request):
     if hasattr(request, 'user') and waffle.flag_is_active(request, EGAP_ADMINS):
         return queryset | RegistrationSchema.objects.filter(name='EGAP Registration').distinct('name')
     else:
-        return queryset
+        return queryset.exclude(name='EGAP Registration')
```

Comment:
This will work, but I think the previous way you had it, with the change below, would be more correct.

Reply:
Thinking more, I don't think this is going to work. I don't see how it's going to prevent other providers from showing the egap registration form when the user has permissions to see that form.